src/farva/pkg/gateway/reverseproxy_nginx.go

```diff
@@ -51,6 +51,18 @@ http {
 {{- end }}
     }
 {{ end }}
+    server {
+        listen {{ .NGINXMetricsPort }};
```

Comment:
The reason this has its own port is because it conflicts with the other server blocks that are listening on the `HTTPListerPort`. This is what we discussed with Jacob because before we had hard coded the port in the `reverseproxy_nginx.go`